src/main/CEO.java

```diff
@@ -0,0 +1,35 @@
+
+
+import java.util.*;
+
+public class CEO {
+    private String name;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public List<Manager> getManagerList() {
+        return managerList;
+    }
+
+    public void setManagerList(List<Manager> managerList) {
+        this.managerList = managerList;
+    }
+
+    public CEO(String name){
+        this.name = name;
+    }
+
+    public List<Manager> managerList = new ArrayList<>();
```

Comment:
\+ za wykorzystanie interfejsu List
ale pola deklarujemy na samym początku klasy:) i ta lista powinna byc prywatna nie powinniśmy mieć do niej dostępu z zewnątrz a jedynie przez zdefiniowany przez Ciebie interfejs czyli w tym wypadku metoda addManager() jest częścią interfejsu Twojej klasy, która za to odpowiada bo tak naprawdę nikogo nie interesuje w czym trzymani są Ci pracownicy. Użytkownik ma mieć jedynie widzieć możliwość zatrudnienia pracownika nie interesuje go w jakis sposób jest to zaimplementowane:)